src/pages/IndexPage.vue

```diff
@@ -123,8 +129,50 @@ const tempData = ref({
   name: '',
   age: '',
 });
+
+onMounted(() => {
+  getData();
+});
+function getData() {
+  axios.get('https://dahua.metcfire.com.tw/api/CRUDTest/a').then((res) => {
+    console.log(res);
+    blockData.value = res.data;
+  });
+}
+function addData(name, age) {
+  axios
+    .post('https://dahua.metcfire.com.tw/api/CRUDTest', {
+      name,
+      age,
+    })
+    .then((res) => {
+      console.log(res);
+      getData();
+      resetForm();
+    });
+}
```

Comment:
這裡可以改寫成 async / await ，並多做錯誤處理看看

```suggestion
async function addData(name, age) {
  try {
     await axios.post('https://dahua.metcfire.com.tw/api/CRUDTest',  tempData.value)
     getData();
     resetForm();
  } catch (error) {
     console.error(error)
  }
  
```